tests/test_flows.py

```diff
@@ -44,14 +44,17 @@ def start_test(self, test: str):
         status(test + " starting")
 
     def check_log(self, entity_or_radio: Union[env.ALEntity, env.Radio], regex: str,
-                  start_line: int = 0) -> bool:
-        '''Verify that the logfile for "entity_or_radio" matches "regex", fail if not.'''
-        return self.wait_for_log(entity_or_radio, regex, start_line, 0.3)
+                  start_line: int = 0, fail_on_mistmatch: bool = True) -> bool:
```

Comment:
I don't understand this change:
changes to `check_log` should be done in a separate commit, with its own rational - as it serves other tests as well.

Reply:
Currently, the `check_log`  function is used as an "assert" statement, 
designed to fail the test when a string was not found in the log.

This change allows testing of the logs without the "assert" part, 
for cases where a missing string is expected or acceptable.